bot/utils/format_song.py

```diff
@@ -0,0 +1,12 @@
+from telegram.utils.helpers import escape_markdown
+
+from bot.models import Song
+
+
+def format_song_markdown(song: Song):
+    return "🎵 `{author} - {song_name}`\n [Spotify]({song_spoti_link}) | [Other]({other_link})".format(
+        song_name=song.name,
```

Comment:
are you sure song name / artist won't have any markdown symbols like `*`, `_`, `~`, etc? consider using "escape_markdown" on name and artist too